src/types.ts

```diff
@@ -196,6 +197,7 @@ export interface LockedBalance {
 export interface GenesisAssetEntry {
 	module: string;
 	data: Record<string, unknown>;
+	schema: any;
```

Comment:
Is it possible to specify the type to be `Schema`?
```suggestion
	schema: Schema;
```